minetest/minetest
# Unify/fix conversion between UTF-8 and wide strings across Minetest and IrrlichtMt

Fixes minetest/irrlicht#216. Depends on a minetest/irrlicht#221.

The UTF-8 / wide string conversion functions in IrrlichtMt are broken on Windows (at least on my machine). Fortunately, Minetest has working ones. This PR moves the working UTF-8 / wide string conversion functions from Minetest to IrrlichtMt so that they can be used by both Minetest and IrrlichtMt. The broken ones have been removed.

I'd be interested in better ideas on how to fix this.

## To do

This PR is a Work in Progress.

- [ ] Move the `iconv` dependency from Minetest to IrrlichtMt

## How to test

Verify that special characters are shown correctly in all contexts and on all platforms, I guess...
\+ Verify that minetest/irrlicht#216 is fixed.
